lambdas/drivingpermitcheck/src/main/java/uk/gov/di/ipv/cri/drivingpermit/api/handler/DrivingPermitHandler.java

```diff
@@ -217,21 +217,44 @@ public APIGatewayProxyResponseEvent handleRequest(
             // Driving Permit Completed Normally
             eventProbe.counterMetric(LAMBDA_DRIVING_PERMIT_CHECK_COMPLETED_OK);
             return ApiGatewayResponseGenerator.proxyJsonResponse(HttpStatusCode.OK, response);
-        } catch (OAuthHttpResponseExceptionWithErrorBody e) {
+        } catch (OAuthErrorResponseException e) {
             // Driving Permit Lambda Completed with an Error
-            LOGGER.error("Encountered error in DCS request : {}", e.getErrorReason());
-            eventProbe.counterMetric(LAMBDA_DRIVING_PERMIT_CHECK_COMPLETED_ERROR);
+
+            // Debug only as Oauth errors appear in the redirect url
+            // This will output the specific error message
+            // Note Unit tests expect server error (correctly)
+            // and will fail if logging is at debug level (during tests)
+            if (LOGGER.isDebugEnabled()) {
```

Comment:
I wonder should this be a param so enabling debug logging doesnt change service functionality